src/views/datasets/DatasetsListView.vue

```diff
@@ -56,6 +67,21 @@ const hasOrganizationFilter = config.website.datasets
 
 const { topicsMainTheme } = useTopicsConf()
 
+const setAccessibilityProperties = inject(
+  'setAccessibilityProperties'
+) as Function
+
+const pageTitle = computed(() => {
```

Comment:
```suggestion
const metaTitle = computed(() => {
```

Comme sur les autres pages.